### PR TITLE
Added Asteroids Dataset

### PR DESCRIPTION
**Pull request by Zignake**
Resolving the Issue: https://github.com/ritwik12/Celestial-bodies-detection/issues/41

### CODE REVIEW

File: hub/examples/image_retraining/label_image.py [hunk 1]
```diff
@@ -69,6 +69,13 @@ def wiki(celestial_object):
         statistics = "\n".join(['-- {}: {}'.format(parameter, value) for parameter, value in req_statistics.items()])
         print("{}\n\n".format(statistics))
         print(wikipedia.WikipediaPage(title='{}'.format(ans)).summary)
+    elif ans == 'asteroid':
```
Comment (ritwik12) [inline hub/examples/image_retraining/label_image.py:72]: @Zignake This will be asteroids

```suggestion
    elif ans == 'asteroids':
```

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@Zignake Thanks!!

**Comment by Zignake** [comment]
> @Zignake Thanks!!

Also, i guess you had forgotten to add the hacktober-fest accepted tag to this PR. Can you do anything about it now?

**Comment by ritwik12** [comment]
@Zignake It was not needed if a PR is merged and I merged your PR in time. I am adding that Label now if it helps somehow.

**Pull request merged**